src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -203,16 +203,27 @@ public function cgetAction(Request $request)
         $listBuilder = $this->getListBuilder($request, $fieldDescriptors, $types);
         $listBuilder->setParameter('locale', $locale);
         $listResponse = $listBuilder->execute();
-        $count = $listBuilder->count();
+
+        $previewImageIds = [];
+        foreach ($listResponse as $listItem) {
+            if ($listItem['previewImageId']) {
```

Comment:
We should check if `$listItem['previewImageId']` isset or fallback with `??` to `null`